src/pages/@owner/account/AccountForm.tsx

```
@@ -152,17 +152,17 @@ export default function Account() {
           />
         }
       />
-      <div className='flex flex-1 flex-col gap-[0.3rem] overflow-y-auto p-[2rem]'>
+      <div className='flex flex-1 flex-col gap-[0.3rem] overflow-y-auto p-[2rem] pt-[6.8rem]'>
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)